### PR TITLE
PARQUET-752: Account for upstream Arrow API changes

### PR DESCRIPTION
**Pull request by wesm**
As soon as ARROW-261/317 are merged, I'll update the thirdparty git SHA so we can get a green build

### CODE REVIEW

File: src/parquet/arrow/writer.cc [hunk 1]
```diff
@@ -45,6 +45,8 @@ using parquet::schema::GroupNode;
 namespace parquet {
 namespace arrow {
 
+namespace BitUtil = ::arrow::BitUtil;
```
Comment (xhochy) [inline src/parquet/arrow/writer.cc]: Didn't know this would work. Why not use `using` here?

Comment (wesm) [inline src/parquet/arrow/writer.cc]: Using doesn't work with namespaces AFAICT

**Comment by wesm** [comment]
Updated the thirdparty SHA. Guess I may have to wait til morning for the OS X builds

**Comment by wesm** [comment]
The externalproject changes in Arrow broke the build, looking into it

**Comment by wesm** [comment]
+1

**Pull request closed without merging**